specs/AppLifecycle/EnvironmentVariables/AppLifecycle Environment Variables.md

```diff
@@ -293,8 +295,6 @@ are listed in the table below:
 | Unpackaged     | Managed  | Read, Write  | Read, Write | Read, Write  |
 | Desktop Bridge | Native   | Read, Write  | Read, Write | Read, Write  |
```

Comment:
Why single out Desktop Bridge?

There are multiple non-UWP types of process -- `RuntimeBehavior=PackagedClassicApp`=DesktopBridge but there's also `RuntimeBehavior=Win32app` and Packaged Services.

SUGGESTION: Drop the table as all the rows have the same answers now